go.mod

```diff
@@ -5,11 +5,11 @@ go 1.13
 require (
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/prometheus/client_golang v0.0.0-20181230203121-fb3d5cb2ad57
-	github.com/prometheus/common v0.0.0-20181218105931-67670fe90761
-	github.com/prometheus/procfs v0.0.0-20190102135031-14fa7590c24d // indirect
-	github.com/sirupsen/logrus v1.2.0
-	github.com/spf13/cobra v0.0.0-20181127133106-d2d81d9a96e2
+	github.com/prometheus/client_golang v1.0.0
```

Comment:
latest is 1.8.0; is it essential to use the older one?

(I don't know if it's important, just asking)

Reply:
I aligned with the current requirements of kubectl, and didn't want to bump too high in a bit. But I thought it would indeed be nice to bump in a follow up commit.